TAF_EasyRest/TestFramework/PageComponents/Administrator/AcceptedCardPageComponent.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedCardPageComponent
+    {
+        private IWebElement _card { get; }
+
+        public AcceptedCardPageComponent(IWebElement _card)
```

Comment:
Parameters should begin with lower case letters. Have you changed your code convention?